org.eclipse.sisu.plexus/src/test/java/org/eclipse/sisu/plexus/ComponentAnnotationTest.java

```diff
@@ -80,12 +83,12 @@ public void testComponentImpl()
         checkBehaviour( "NamedPrototypeA" );
         checkBehaviour( "DescribedA" );
 
-        assertFalse( replicate( getComponent( "DefaultA" ) ).equals( getComponent( "NamedA" ) ) );
-        assertFalse( replicate( getComponent( "DefaultA" ) ).equals( getComponent( "PrototypeA" ) ) );
-        assertFalse( replicate( getComponent( "DefaultA" ) ).equals( getComponent( "DescribedA" ) ) );
-        assertFalse( replicate( getComponent( "Simple" ) ).equals( getComponent( "DefaultA" ) ) );
-        assertFalse( replicate( getComponent( "Simple" ) ).equals( getComponent( "Simple2" ) ) );
-        assertFalse( replicate( getComponent( "Simple" ) ).equals( getComponent( "Simple3" ) ) );
+        assertNotEquals(replicate(getComponent("DefaultA")), getComponent("NamedA"));
```

Comment:
Something to consider for the future, [org.hamcrest.MatcherAssert.assertThat](https://examples.javacodegeeks.com/java-development/core-java/junit/junit-assertthat-example/) can increase readability, esp with the **is** and **equalTo** methods

`assertThat("Actual", is(equalTo("Expected"))'